src/utils/__tests__/internalLinks-test.js

```diff
@@ -116,61 +131,85 @@ describe('isNarrowLink', () => {
   }
 });
 
-describe('getLinkType', () => {
-  test('links to a different domain are of "non-narrow" type', () => {
-    expect(getLinkType(new URL('https://google.com/some-path'), realm)).toBe('non-narrow');
-  });
+// TODO: Combine / dedupe with "getNarrowFromLink (more)" tests, below
+describe('getNarrowFromLink', () => {
```

Comment:
Perhaps give this a name like "getNarrowFromLink (part 1)" — something that doesn't suggest that this is all of the getNarrowFromLink tests.